privacy/whonix.md

```diff
@@ -24,30 +24,25 @@ while the workstation is used for making AppVMs.
 
 ## Getting Started with Whonix
 
-To install Whonix, you must have a working Qubes machine already.
```

Comment:
I am not sure that is obvious, but if others agree it is, by all means feel free to remove that sentence.